src/resources/pandoc/datadir/init.lua

```diff
@@ -1789,7 +1791,12 @@ _quarto = {
    projectOffset = projectOffset,
    file = {
       read = read_file,
-      write = write_file,
+      write = function(path, contents) 
+         return write_file(path, contents, "ab") 
```

Comment:
Should it be instead ? 
```suggestion
         return write_file(path, contents, "wb") 
```